src/main/java/marquez/api/model/LineageEvent.java

```diff
@@ -0,0 +1,219 @@
+package marquez.api.model;
```

Comment:
Mind moving the `LineageEvent` class to the [`service.models`](https://github.com/MarquezProject/marquez/tree/main/src/main/java/marquez/service/models) pkg to keep things consistent with how the resource layer currently interacts with the service layer?